api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java

```diff
@@ -580,32 +580,60 @@ private void enableVitals() {
 
         }
 
-        extensions.add(visitAction(Extensions.VITALS_CAPTURE_VISIT_ACTION,
-                "mirebalais.task.vitals.label",
-                "fas fa-fw fa-heartbeat",
-                "link",
-                enterSimpleHtmlFormLink(PihCoreUtil.getFormResource("vitals.xml")),
-                null,
-                and(sessionLocationHasTag("Vitals Location"),
-                        or(and(userHasPrivilege(  PihEmrConfigConstants.PRIVILEGE_TASK_EMR_ENTER_VITALS_NOTE), patientHasActiveVisit()),
-                                userHasPrivilege(  PihEmrConfigConstants.PRIVILEGE_TASK_EMR_RETRO_CLINICAL_NOTE),
-                                and(userHasPrivilege(  PihEmrConfigConstants.PRIVILEGE_TASK_EMR_RETRO_CLINICAL_NOTE_THIS_PROVIDER_ONLY), patientVisitWithinPastThirtyDays(config))))));
+        if(config.getCountry().equals(ConfigDescriptor.Country.PERU)){
```

Comment:
Is the only reason for introducing the different block for Peru here is because you want to change the label to "Triaje", and we can't update the Spanish translation because they want a different label in Mexico?  Ugh, but, yeah, that may be the easiest way to do this at this point.  I might have just set the name as a variable at the top of the enableVitals class based on country and then use a single block to add the extension to make it clear that the name is the only thing that is different. 

One of the last main things to do in the "Config Initiative" project is to refactor this file out into something that is easier/clearer for different countries to specify there own configurations.  This is another reason for us to push for us to do this in the near future.

Reply:
Hmm... ah, yes, thank makes sense, I'd have to think a little bit about the potential answers.    The issue is that we want a different name between the Peru and Mexico implementations, right?

I started writing out a couple options, but think I have one I like: moving the encounter types into a separate messages.properties file and then overriding them in Peru.  I need to sign off for the night now, but could look more into this tomorrow.